lib/scidata/caltech101.ex

```diff
@@ -128,6 +129,15 @@ defmodule Scidata.Caltech101 do
       |> Nx.new_axis(-1)
       |> Nx.equal(Nx.tensor(Enum.to_list(1..102)))
 
+  ## Options.
+
+    * `:base_url` - optional. Dataset base URL.
+      Defaults to `"https://s3.amazonaws.com/fast-ai-imageclas/"`
+    * `:dataset_file` - optional. Dataset filename.
+      Defaults to `"caltech_101.tgz"`
+    * `:cache_dir` - optional. Cache directory.
```

Comment:
What do you think about calling this option `data_path`? Tensorflow datasets [calls it `images_dir_path`](https://github.com/tensorflow/datasets/blob/master/tensorflow_datasets/image_classification/caltech.py#L95) for e.g. Caltech 101 dataset, and maybe users will think of this option as the "path to the data" rather than a cache. Either is fine, I'm curious about your opinion.

Reply:
From my perspective, calling something `cache` means I can safely delete it if I don't need the cache anymore. And if I need it in the future, the code should be able to re-create that again without causing issues. Whereas calling it `data_path`, the users might think the code would expect the files to exist there.